docs/data-sources.md

```diff
@@ -32,7 +32,7 @@ The Gecko Profiler records marker data, but it doesn't include all of the marker
 
 TaskTracer provides a way to trace the correlation between different Runnables across threads and processes. These Runnables are defined by `mozilla::Runnable`, and represent a task which can be dispatched to a thread for execution. Unlike sampling based profilers, TaskTracer can tell you where a Runnable is dispatched from, what its original source was, how long it waited in the event queue, and how long it took to execute.
 
-Source Events are usually some kinds of I/O events we're interested in, such as touch events, timer events, network events, etc. When a source event is created, TaskTracer records the entire chain of Runnables as they are dispatched to different threads and processes. It records latency, execution time, etc. for each Runnable that chains back to the original source event.
+Source Events are usually some kinds of I/O events we're interested in, such as touch events, timer events, network events, etc. When a source event is created, TaskTracer records the entire chain of Runnables as they are dispatched to different threads and processes. It records latency, execution time, etc. for each Runnable that chains back to the original source event. There was an initial prototype of integrating this information into perf.html, but it has since been removed and is available under the git tag [tasktracer-removal](https://github.com/devtools-html/perf.html/releases/tag/tasktracer-removal).
```

Comment:
I need to add this tag after I land the PR.

Reply:
I think you need to add the tag the version from just _before_ you land. You can do that before or after but I think it's easier to do it before :)